UnitsNet/CustomCode/UnitClasses/Length.extra.cs

```diff
@@ -37,6 +35,39 @@ public static Length FromFeetInches(double feet, double inches)
         {
             return FromInches((FeetToInches * feet) + inches);
         }
+
+        public static Speed operator/ (Length length, TimeSpan timeSpan)
+        {
+            return Speed.FromMetersPerSecond(length.Meters / timeSpan.TotalSeconds);
+        }
+
```

Comment:
Thanks! I think I will try ReSharper again, was a couple of years since I tried it and these are things were it really shines (which I didn't thought I needed back then).